modules/ROOT/pages/changelog.adoc

```diff
@@ -4,6 +4,91 @@
 
+== 6.4 - 2023-03-15
+
+=== Added
+* New `tree` component that can be used in dialog body panel.
+* renderUI property in the `Theme` type can now return a `Promise<RenderResult>` instead of `RenderResult`.
+* New `isEditable` API to `editor.selection` that returns true or false if the current selection is editable.
+* New `isEditable` API to `editor.dom` that returns true or false if the specified node is editable.
+* New `setText` and `setIcon` methods added to menu button and toolbar button apis.
+* New `highlight_on_focus` option which enables highlighting the content area on focus.
+* New `fontsizeinput` toolbar item which allows the user to set the size via input and also increase and decrease it with `+` and `-` buttons.
+* Added `skipFocus` option to the `ToggleToolbarDrawer` command to preserve focus.
+* New `font_size_input_default_unit` option allows entry of numbers without a unit in `fontsizeinput`. They are then parsed as the set unit. If `font_size_input_default_unit` is not set the default is `pt`.
+* New `group` and `togglebutton` in view.
+* New `togglebutton` in dialog footer buttons.
+* Added `toggleFullscreen` to dialog API.
+* New `text-size-increase` and `text-size-decrease` icons.
+* New `xss_sanitization` option to allow disabling of XSS sanitization.
+* Added the top right close button of modal dialogs to the tabbing order. The 'x' button in these dialogs can now be accessed using keyboard navigation.
+* New `ui_mode` option for editor in scrollable containers support.
+
+=== Improved
+* Direct invalid child text nodes of list elements are now wrapped in list item elements.
+* Templates are now be parsed before preview and insertion to make preview consistent with inserted template content and prevent XSS.
+* Pressing backspace on an empty line now preserves formatting on the previous empty line.
+* Pressing enter inside the `inputfontsize` input field now moves focus back into the editor content.
+* Drag and drop events for elements with a `contenteditable="false"` attribute now includes target element details.
+* Updated focus, active, and enabled colors of UI buttons for improved contrast against the UI color.
+
+=== Changed
+* The `link` plugins context menu items no longer appears for links that include elements with a `contenteditable="false"` attribute.
+* The formatting of elements with a `contenteditable="false"` attribute are no longer cloned to new cells when new table rows are created.
+* Changed the color of `@dialog-table-border-color`, and added right padding to the first cell of dialog table.
+* The sidebar element now has the accessibility role `region` when visible and the accessibility role `presentation` when hidden.
+* The `tox-custom-editor` class now has a border highlight when it is selected.
+
+=== Fixed
+* Add support for navigating inside the tree component using arrow keys and shift key.
```

Comment:
This should be moved to `Added` section

Reply:
@tiny-ben-tran,

Is it safe to assume you want the equivalent change made to TinyMCE’s `changelog.md`?

Reply:
Yes. I left the same comments in two different places. Sorry about that

Reply:
Moved.